include/fiction/algorithms/physical_design/graph_oriented_layout_design.hpp

```diff
@@ -752,6 +842,7 @@ class graph_oriented_layout_design_impl
                 }
             }
 
+            // check if timeout is reached or solution found
             timeout_limit_reached =
                 std::none_of(ssg_vec.cbegin(), ssg_vec.cend(), [](const auto& ssg) { return ssg.frontier_flag; });
```

Comment:
This doesn't seem to check for the time at all. Is it correctly named?

Reply:
It checks if any of the frontiers is still active, otherwise, timeout is reached already.
Later, it is checked whether the set timeout is reached.